source/anonymize/index.rst

```diff
@@ -0,0 +1,1309 @@
+Anonymize
+=========
+
+Requirements (on-premise)
+-------------------------
+
+============ =========== ===========
+GLPI Version Minimum PHP Recommanded
+============ =========== ===========
+10.0.x       8.1         8.2
+============ =========== ===========
+
+Install the plugin
+------------------
+
+.. Note::
+   A `basic licence <https://services.glpi-network.com/#offers>`__ (or higher) is required. This plugin is also available from the `Cloud <https://glpi-network.cloud/fr/>`__.
+
+-  Go to the marketplace. Download and install the “**Anonymize**”   plugin.
+
+.. figure:: images/Anonymize-1.png
+   :alt: Install the plugin
+
+Choice of profile
+-----------------
+
+Anonymisation works by profile. There are 4 profiles already created.
+When you select a profile, a certain amount of data is already set. They can, of course, be modified.
+
+-  Go to **administration** > **anonymization profile** and select the profile best suited to your needs:
+
+The 4 profiles you will find are :
+
+-  Email,
+-  Entity,
+-  Computer
+-  User profile
```

Comment:
Not a big deal, but in English we don't typically add commas or semicolons to lists like that. We don't typically add periods either unless the item is a full sentence.
```suggestion
-  Email
-  Entity
-  Computer
-  User profile
```